grails-app/views/layouts/main.gsp

```diff
@@ -100,6 +100,17 @@
 
     </style>
 
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-54091417-4', 'auto');
```

Comment:
Yeah, couldn't be arsed doing that here for now (espicially given that it's not a planned task).  Just means all environment analytics will get lumped together I guess.